packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -209,6 +209,12 @@ export class SwirlFormControl {
 
     const isSelect = this.inputEl.tagName === "SWIRL-SELECT";
 
+    const tooltip: string | null = (() => {
```

Comment:
Issue: I think this should be a prop.

Reply:
Done. This makes the Question regarding the IIFE obsolete right?

Reply:
Yes